apps/server/src/controllers/userController.ts

```diff
@@ -104,3 +111,62 @@ export const registerUser = async (req: Request, res: Response) => {
     })
   }
 }
+
+//===================LOGIN-USER========================\\
+export const loginUser = async (req: Request, res: Response) => {
+  try {
+    const { email, password } = req.body
+
+    const validationResult = loginSchema.safeParse(req.body)
+
+    if (!validationResult.success) {
+      return res.status(400).json({
```

Comment:
I think its best we use the error code variables declared in httpStatusCode.ts so that our codes will b uniform throughout. or what do you think?